src/AWS.Deploy.Orchestration/ServiceHandlers/AWSElasticBeanstalkHandler.cs

```diff
@@ -93,7 +93,7 @@ private T GetOrCreateNode<T>(object? json) where T : new()
         {
             try
             {
-                return JsonSerializer.Deserialize<T>(json?.ToString() ?? string.Empty);
+                return JsonSerializer.Deserialize<T>(json?.ToString() ?? string.Empty, new JsonSerializerOptions { ReadCommentHandling = JsonCommentHandling.Skip });
```

Comment:
Are we overwriting and effectively deleting the user's comments when we write the file back out?

If so, should we not do that?

Reply:
Yes we are, but it shouldn't matter since we are not replacing the manifest file in the user's project. The updated file is only added to the .net zip deployment bundle.